data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/server/PeerForwarderHttpService.java

```diff
@@ -46,16 +47,19 @@ public class PeerForwarderHttpService {
     private final PeerForwarderCodec peerForwarderCodec;
     private final Timer serverRequestProcessingLatencyTimer;
     private final Counter recordsReceivedFromPeersCounter;
+    private final AcknowledgementSetManager acknowledgementSetManager;
 
     public PeerForwarderHttpService(final ResponseHandler responseHandler,
                                     final PeerForwarderProvider peerForwarderProvider,
                                     final PeerForwarderConfiguration peerForwarderConfiguration,
                                     final PeerForwarderCodec peerForwarderCodec,
+                                    final AcknowledgementSetManager acknowledgementSetManager,
                                     final PluginMetrics pluginMetrics) {
         this.responseHandler = responseHandler;
         this.peerForwarderProvider = peerForwarderProvider;
         this.peerForwarderConfiguration = peerForwarderConfiguration;
         this.peerForwarderCodec = peerForwarderCodec;
+        this.acknowledgementSetManager = acknowledgementSetManager;
```

Comment:
How is this used ?

Reply:
This will be used later to send acknowledgements of events that are sent to peers.